src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicTlsSecret.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Net.Quic;
 internal sealed class MsQuicTlsSecret : IDisposable
 {
     private static readonly string? s_keyLogFile = Environment.GetEnvironmentVariable("SSLKEYLOGFILE");
-    private static readonly FileStream? s_fileStream = s_keyLogFile != null ? File.Open(s_keyLogFile, FileMode.Append, FileAccess.Write) : null;
+    private static readonly FileStream? s_fileStream = s_keyLogFile != null ? File.Open(s_keyLogFile, FileMode.Append, FileAccess.Write, FileShare.ReadWrite) : null;
```

Comment:
Is this supposed to be guarded by a DEBUG as well?

Reply:
whole class is `#if DEBUG` (line 4, not in the diff)
The Quick code is more complicated because of MsQuic in the middle.